tests/test_util_file.expected.Darwin

```diff
@@ -0,0 +1,35 @@
+OK: json_object_from_fd(./valid.json)={ "foo": 123 }
```

Comment:
ack;
this will have to be done via a preprocessor `#define` at build time in `.travis.yml` ; or maybe via a special `configure` rule ;
because `libjson-c` is built first and that would link to `strerror`

i also thought about renaming `strerror`
wasn't sure what your thoughts/input would be